### ISSUE TITLE
Hi, I am new in this field.

### ISSUE THREAD

**Issue by jackyluo-learning** [vudaoanhtuan/Transformer#1]
Hi! I am new in this field and trying to learn something. Is it the evolved transformer? The one that applied the NAS method?

**Comment by vudaoanhtuan** [comment]
Yes, I re-implement it from [tensor2tensor](https://github.com/tensorflow/tensor2tensor/blob/master/tensor2tensor/models/evolved_transformer.py) lib, but I haven't tested it yet

**Comment by jackyluo-learning** [comment]
Thanks a lot, recently I am working on Evolved Transformer, and I feel a bit struggle of it, so I am looking for resources, but it is hard to find more since this has just come out few months ago. Could you recommend some resources to me? Thanks

**Comment by vudaoanhtuan** [comment]
You should read its [paper](https://arxiv.org/pdf/1901.11117) and its implementation in [tensor2tensor](https://github.com/tensorflow/tensor2tensor/blob/master/tensor2tensor/models/evolved_transformer.py) lib.
I think it is not popular so there are not many tutorials on web
